src/util/xml.cpp

```diff
@@ -19,6 +19,13 @@ double XmlParse::selectNodeDouble(const QDomNode& nodeHeader,
     return selectNode(nodeHeader, sNode).toElement().text().toDouble(ok);
 }
 
+bool XmlParse::selectNodeBool(const QDomNode& nodeHeader,
+        const QString& sNode,
+        bool* ok) {
```

Comment:
unused parameter

Reply:
fixed in d4582f69ef52b7252f43822a8d1e8ccbdc8463e5